automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -80,18 +80,21 @@ trait GPAllocUtils extends BillingFixtures with LeonardoTestUtils {
             hermioneAuthToken
           )
       )
-      _ <- IO(releaseGPAllocProject(project.value, hermioneCreds))
-      _ <- IO(logger.info(s"Billing project released: ${project.value}"))
+      releaseProject <- IO(releaseGPAllocProject(project.value, hermioneCreds)).attempt
```

Comment:
Don't fail the test if release fails -- gpalloc will clean up unreleased projects after 2 hours